_typos.toml

```diff
@@ -0,0 +1,2 @@
+[files]
+extend-exclude = ["test_p256_account.cairo"]
```

Comment:
Why are we excluding this file?

Reply:
Because your tool that checks for syntax consider I don't spell some hexa value wrong and I do not know any other ways to pass the associated test.